wordcount/wordcount.go

```diff
@@ -17,10 +21,27 @@ func mapFunc(input []byte) (result []mapreduce.KeyValue) {
 	//	Map should also make words lower cased:
 	//		strings.ToLower(string)
 
-	/////////////////////////
-	// YOUR CODE GOES HERE //
-	/////////////////////////
-	return make([]mapreduce.KeyValue, 0)
+	var happyMap = make([]mapreduce.KeyValue, 0)
+	str := strings.ToLower(string(input[:]) + " ")
+	word := ""
```

Comment:
O valor zero de string já é uma string vazia. 
http://stackoverflow.com/questions/12703243/golang-what-is-the-zero-for-string